pyrasterframes/src/main/python/tests/RasterFunctionsTests.py

```diff
@@ -96,6 +96,15 @@ def test_general(self):
 
         df.first()
 
+    def test_st_geometry_from_struct(self):
+        from pyspark.sql import Row
+        from pyspark.sql.functions import struct
+        df = self.spark.createDataFrame([Row(xmin=0, ymin=1, xmax=2, ymax=3)])
+        df.select(st_geometry(struct(df.xmin, df.ymin, df.xmax, df.ymax)).alias('geom'))
```

Comment:
I hate that you have to `struct` an already named `Row`....

Reply:
hm maybe you could just pass the whole row in ? idk.

Reply:
@vpipkt No, there's a bug in PySpark + Python 3.6 that makes field order non-deterministic.